README.md

```diff
@@ -113,17 +113,25 @@ var node = new ipfsMDAG.DAGNode([<data>, <[links]>])
 
 ### DAGService
 
-##### add
+##### put
 
 > stores the node
 
+##### putStream
+
+> stores nodes using a writable pull-stream
+
 ##### get
 
 > fetches a node by its multihash
 
-##### getRecursive
+##### getStream
+
+> fetches a node as a pull-stream
+
+##### getRecursiveStream
 
-> fetches a node and all of its links (if possible)
+> fetches a node and all of its links (if possible) as pull-stream
```

Comment:
but we are not using even one currently :/ and I don't like increasing the api surface